cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/CassandraDialect.java

```diff
@@ -210,12 +236,61 @@ public void removeTuple(EntityKey key, TupleContext tupleContext) {
 
 	@Override
 	public Association getAssociation(AssociationKey key, AssociationContext associationContext) {
-		return null; // TODO
+
+		String table = key.getTable();
+		Table tableMetadata = provider.getMetaDataCache().get( key.getTable() );
```

Comment:
really? It's mutated only at setup time (single threaded), then read (maybe concurrently) only at query execution time. Have I misunderstood the concurrency model? I don't see the case where it's going to get concurrent mutations and reads.